kernel-spark/src/main/java/org/apache/spark/sql/delta/DeltaDsv2EnableConf.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (2025) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.delta;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import org.apache.spark.internal.config.ConfigEntry;
+import org.apache.spark.sql.internal.SQLConf;
+import scala.jdk.javaapi.CollectionConverters;
+
+/**
+ * Configuration for enabling DataSourceV2 mode in Delta Lake.
+ *
+ * <p>This is a standalone configuration class that does not depend on DeltaSQLConf, allowing for
+ * independent evolution of the DSv2 feature.
+ *
+ * <p>Note: This class uses {@code package org.apache.spark.sql.delta} (not {@code
+ * io.delta.kernel.spark}) to access Spark's internal config API ({@link SQLConf#buildConf}), which
+ * is only accessible from {@code org.apache.spark.*} packages.
+ */
+public class DeltaDsv2EnableConf {
```

Comment:
Lets start calling everything just v2 instead of DSv2, just like we have defined sparkv1 and sparkv2. 
also even the V1 connector already uses a lot of the DSv2 APIs.. so its confusing.